js/app.js

```diff
@@ -194,6 +250,9 @@
    *     event, 'light-green'
    */
 
+   btn13.addEventListener(CLICK, function(event) {
+    setMyLightClass.bind(btn13).call(this, event, 'light-green');});
```

Comment:
this can be simplified. if bind returns a function, try invoking right after the bind function is invoked.